modules/bmp/image_loader_bmp.cpp

```diff
@@ -241,9 +241,8 @@ Error ImageLoaderBMP::load_image(Ref<Image> p_image, FileAccess *f,
 				case BI_CMYKRLE8:
 				case BI_CMYKRLE4: {
 					// Stop parsing
-					ERR_EXPLAIN("Compressed BMP files are not supported: " + f->get_path());
 					f->close();
-					ERR_FAIL_V(ERR_UNAVAILABLE);
+					ERR_FAIL_V_MSG(ERR_UNAVAILABLE, "Compressed BMP files are not supported: " + f->get_path() + ".");
```

Comment:
Might crash here, I think `f->close()` nullifies the pointer.

Reply:
Oh yeah, fair, get the path and separate out into var first i guess?

Reply:
Yes, you can save the path first and use it here. This `case` is an error condition in itself already so the performance impact is not a problem.